packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -438,7 +432,7 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     positionRef.current = { x: event.clientX, y: event.clientY };
 
     if (popperRef.current) {
-      popperRef.current.scheduleUpdate();
+      popperRef.current.forceUpdate();
```

Comment:
In a demo we use `update` (former `scheduleUpdate`) (https://github.com/mui-org/material-ui/pull/21761/files?short_path=419e190#diff-da17cb0c3c0e721ba02d727f35d997c417ec1b540cc2aa9e64ecdb9681389381L18-R18) but here we're forcing a sync update. What motivated this change?

Reply:
It's required to have the test pass in sync mode, otherwise we need to in it.

Reply:
Then the test should be updated. We shouldn't regress in behavior just to make our life easier.

Reply:
Ok https://github.com/popperjs/popper-core/blob/264e2f31b1b0a765504aa4d4ee85ca0b0e319a9e/src/createPopper.js#L160-L164